tools/release.rb

```diff
@@ -101,8 +101,12 @@ def self.generate_release_version(release_type)
 
   private
 
-  def remove_old_releases
-    FileUtils.rm Dir.glob(File.join(Machinery::ROOT, "package/*"))
+  def remove_old_releases(skip_cleanup)
```

Comment:
I think it is Ok, since we only clean up the tmp directory there which should be always fine. The reason why we added skip_cleanup was to preserve the packages for our integration test which should still work.